### PR TITLE
Re-implement the header without depending on MWC

### PR DESCRIPTION
**Pull request by beverloo**
This reduces our JavaScript bundle size by 15KB, while adding 7KB of CSS. The large gains will be made when dropping our dependencies on MWC altogether.

Material Web uses SASS, so in order to be able to import styles we need to support that too. This implicitly already was the case because MWC used it internally, this makes it explicit. The `MiniCssExtractPlugin` dependency is no longer needed, and has been removed.

### CODE REVIEW

**Comment by beverloo** [comment]
Live on https://llaminator.peter.sh/ by the way. There shouldn't be any visual difference.

File: src/components/llama-header.ts [hunk 1]
```diff
@@ -15,14 +15,29 @@
...
 
 import { LitElement, html } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { customElement, property } from 'lit/decorators.js';
 
 @customElement('llama-header')
 export class LlamaHeader extends LitElement {
+  @property({ type: String }) title: string;
+
+  constructor() {
```
Comment (glennhartmann) [inline src/components/llama-header.ts:24]: I don't think this constructor is necessary - you can assign a default value to the `title` property when it's declared

File: package.json [hunk 2]
```diff
@@ -15,8 +15,11 @@
     "copy-webpack-plugin": "^10.1.0",
     "css-loader": "^6.5.1",
     "css-minimizer-webpack-plugin": "^3.2.0",
+    "extract-loader": "^5.1.0",
+    "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.5.0",
-    "mini-css-extract-plugin": "^2.4.5",
+    "sass": "^1.45.0",
```
Comment (glennhartmann) [inline package.json:21]: something else I've never heard of... how many more meta-languages should I know about?

**Review by glennhartmann — APPROVED** [review]

File: src/components/llama-header.ts (hunk 1 above)
Comment (beverloo) [reply src/components/llama-header.ts:24]: Fair, I found warnings against this when I implemented it, but I guess it's OK now.. https://lit.dev/docs/v1/components/properties/#initialize-property-values-when-using-decorators

File: package.json (hunk 2 above)
Comment (beverloo) [reply package.json:21]: You're in for a treat: https://roadmap.sh/frontend

**Pull request merged**